tests/test_modeling_encoder_decoder.py

```diff
@@ -776,6 +776,24 @@ def get_pretrained_model(self):
     def test_encoder_decoder_model_shared_weights(self):
         pass
 
+    @slow
+    def test_bert2gpt2_summarization(self):
+        model = EncoderDecoderModel.from_pretrained("patrickvonplaten/bert2gpt2-cnn_dailymail-fp16")
```

Comment:
Hi, @patrickvonplaten Is this a mistake: `EncoderDecoderModel` here? If so, maybe we should check the result once fixed.

Reply:
Nono that is correct I think, why would it be a mistake?

Reply:
Hi, Sorry, I made a mistake (thought it it in Flax test file, but it is actually for PyTorch test).